newsfragments/1705.bugfix

```diff
@@ -0,0 +1 @@
+Speed up processing still images by reducing the number of file reads per image
```

Comment:
`dials.stills_process`: improve processing performance by preventing re-reading of image data

?